ext/rugged/rugged_commit.c

```diff
@@ -662,11 +693,77 @@ static VALUE rb_git_commit_extract_signature(int argc, VALUE *argv, VALUE self)
 	return ret;
 }
 
+/*
+ *  call-seq:
+ *    Commit.create_buffer(repository, data = {}) -> commit
```

Comment:
This should say `-> String`. But maybe `create_buffer` is not the ideal name here. `Commit#compose` maybe?